pyproject.toml

```diff
@@ -53,13 +53,13 @@ build-backend = 'setuptools.build_meta'
     enabled = true
 
   [ tool.gilesbot.circleci_artifacts.figure_report ]
-    url = "py310-figure/figure_test_images/fig_comparison.html"
-    message = "Click details to see the figure test comparisons, for figure."
+    url = ".tmp/py310-figure/figure_test_images/fig_comparison.html"
```

Comment:
Should these urls really be to `.tmp` locations?

Reply:
They are .tmp so that they don't pollute your local directory if you run this locally.

Reply:
That being said, you shouldn't actually need them here. Giles will match on any substring.

Reply:
I can undo it?